crates/core/tedge_mapper/Cargo.toml

```diff
@@ -37,8 +37,10 @@ csv = "1.1"
 download = { path = "../../common/download" }
 flockfile = { path = "../../common/flockfile" }
 futures = "0.3"
+logged_command = { path = "../../common/logged_command" }
 mockall = "0.11"
 mqtt_channel = { path = "../../common/mqtt_channel" }
+plugin_sm = { path = "../plugin_sm" }
```

Comment:
Why the `plugin_sm` crate has to be moved out of the agent?

Reply:
because i could not import the agent code into the mapper